File: C11/백준/Bronze/33612. 피갤컵/피갤컵.c
#include <stdio.h>

int main(void) {
	
	int N;
	scanf("%d", &N);
	
	int year = 2024;
	int month = 8;
	
	month += (N - 1) * 7;
	while(month > 12) {
		month -= 12;
		year++;
	}
	
	printf("%d %d", year, month);
	
	return 0;
}

File: Java/백준/Bronze/33612. 피갤컵/피갤컵.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine()); //N회 피갤컵

        int year = 2024; //1회 피갤컵
        int month = 8; //1회 피갤컵

        month += (N - 1) * 7; //7개월 주기로 열림
        while (month > 12) {
            month -= 12;
            year++;
        }

        System.out.print(year + " " + month);
    }
}